functional-tests/src/test/java/org/infinispan/online/service/caching/CachingServiceTest.java

```diff
@@ -29,30 +32,60 @@
    @ArquillianResource
    URL restService;
 
+   @Named("caching-service-app-memcached")
+   @ArquillianResource
+   URL memcachedService;
+
    @ArquillianResource
    OpenShiftClient openShiftClient;
 
    ReadinessCheck readinessCheck = new ReadinessCheck();
    HotRodTester hotRodTester = new HotRodTester();
    RESTTester restTester = new RESTTester();
+   MemcachedTester memcachedTester = new MemcachedTester();
 
    @Before
    public void before() {
       readinessCheck.waitUntilAllPodsAreReady(openShiftClient, 30, TimeUnit.SECONDS);
    }
 
    @Test
-   public void should_default_cache_be_accessible_via_hot_rod() throws IOException {
-      hotRodTester.testBasicEndpointCapabilities(hotRodService);
+   @Ignore // TODO: remove when https://issues.jboss.org/browse/ISPN-8536 is resolved
+   public void should_not_blow_up_because_of_oom() {
+      hotRodTester.testPutPerformance(hotRodService, 60, TimeUnit.SECONDS);
+   }
+
+   @Test
+   public void rest_put_get() throws IOException {
+      restTester.putGetRemoveTest(restService);
    }
 
    @Test
-   public void should_not_blow_up_because_of_oom() {
-      hotRodTester.testPutPerformance(hotRodService, 60, TimeUnit.SECONDS);
+   public void hotrod_put_get() {
+      hotRodTester.putGetTest(hotRodService);
    }
 
    @Test
-   public void should_default_cache_be_accessible_via_REST() throws IOException {
-      restTester.testBasicEndpointCapabilities(restService);
+   public void memcached_put_get() {
+      memcachedTester.putGetRemoveTest(memcachedService);
+   }
+
+   // The eviction can not be turned off on caching service
+   // The test puts entries to the cache until one is evicted
+   @Ignore // TODO: remove when https://issues.jboss.org/browse/ISPN-8536 is resolved
+   @Test(timeout = 600000)
+   public void eviction_test() {
```

Comment:
This can also be uncommented.

Also the naming can be better so that the comment above would not be necessary. Maybe something like `should_put_entries_until_first_one_gets_evicted` or `should_evict_entries_before_reaching_timeout`. Also be careful with those timeouts... our CI isn't very fast and I bet it's much slower than your laptop.